doc-site/.vitepress/theme/customElementsManifest.store.ts

```diff
@@ -100,17 +100,25 @@ const getComponentClassDeclaration = (componentName: string): CustomElementDecla
  * @returns en komponent med angitt navn eller undefined om komponenten ikke finnes
  */
 const getModuleByName = (name: string): Module | undefined => {
```

Comment:
jeg må si at jeg ikke forstår hvorfor vi må skrive mer kode her for å oppnå samme resultat? 
bortsett fra en sjekk om !componentNames.value.includes(name) på linje 121, syns jeg utvidet kode er unødvendig